snmp/datadog_checks/snmp/snmp.py

```diff
@@ -125,15 +126,10 @@ def _get_instance_name(self, instance):
         else:
             return None
 
-    def discover_instances(self):
-        # type: () -> None
+    def discover_instances(self, interval):
+        # type: (int) -> None
         config = self._config
 
-        if config.ip_network is None:
-            raise RuntimeError("Expected config.ip_network to be set to start discovery")
```

Comment:
We don't need this check?

Reply:
It's an artefact from #5785: the check is already done by `config.network_hosts()` now (called further down this method):

https://github.com/DataDog/integrations-core/blob/37096273b199423a2897ad5e3710b5de046c3614/snmp/datadog_checks/snmp/config.py#L287-L290